util-core/src/main/scala/com/twitter/util/Batcher.scala

```diff
@@ -0,0 +1,20 @@
+package com.twitter.util
+
+/** Provides a clean, lightweight interface for working with batched Futures */
+private[util] class Batcher[In, Out](
+  executor: BatchExecutor[In, Out]
+)(
+  implicit timer: Timer
+) extends Function1[In, Future[Out]] { batcher =>
+  /** Enqueues requests for a batched Future */
```

Comment:
Also, you are speaking to me in regex here I guess?  You want me to fully qualify all my references to com.twitter.util.Future?  Or just the one on this line?  And only in comments or in comments and code?